### ISSUE TITLE
Launcher not starting up at all. (Only Console and Splash are visible)

### ISSUE THREAD

**Issue by edenprojectde** [ATLauncher/ATLauncher#364]
I get this when Launching the Launcher after updating 20 Minutes ago when i started my PC after 3 days or so.
Im on Linux MX (Ubuntu Deriviant)

https://pastebin.com/93XjNs2p

**Comment by RyanTheAllmighty** [comment]
Yeah there's an issue in the code that's attempting to rename directories.

I'm working on it and hope to have a fix out soon

**Comment by RyanTheAllmighty** [comment]
Released with version 3.3.0.3

**Comment by edenprojectde** [comment]
```
[29/7/2019 13:02:23 PM] Organising filesystem
[29/7/2019 13:02:23 PM] Error renaming directory
[29/7/2019 13:02:23 PM] java.nio.file.DirectoryNotEmptyException: /home/Ben/Games/Minecraft/ATLauncher/faileddownloads
	at sun.nio.fs.UnixCopyFile.move(UnixCopyFile.java:445)
	at sun.nio.fs.UnixFileSystemProvider.move(UnixFileSystemProvider.java:262)
	at java.nio.file.Files.move(Files.java:1395)
	at com.atlauncher.FileSystem.renameDirectory(FileSystem.java:145)
	at com.atlauncher.FileSystem.renameDirectories(FileSystem.java:112)
	at com.atlauncher.FileSystem.organise(FileSystem.java:79)
	at com.atlauncher.App.main(App.java:250)
[29/7/2019 13:02:23 PM] Error renaming directory
[29/7/2019 13:02:23 PM] java.nio.file.DirectoryNotEmptyException: /home/Ben/Games/Minecraft/ATLauncher/configs
	at sun.nio.fs.UnixCopyFile.move(UnixCopyFile.java:445)
	at sun.nio.fs.UnixFileSystemProvider.move(UnixFileSystemProvider.java:262)
	at java.nio.file.Files.move(Files.java:1395)
	at com.atlauncher.FileSystem.renameDirectory(FileSystem.java:145)
	at com.atlauncher.FileSystem.renameDirectories(FileSystem.java:113)
	at com.atlauncher.FileSystem.organise(FileSystem.java:79)
	at com.atlauncher.App.main(App.java:250)
[29/7/2019 13:02:23 PM] ATLauncher Version: 3.3.0.4
[29/7/2019 13:02:23 PM] Operating System: Linux
[29/7/2019 13:02:23 PM] java.lang.NullPointerException
	at com.atlauncher.utils.Java.getMinecraftJavaVersion(Java.java:93)
	at com.atlauncher.utils.Java.getMinecraftJavaVersionNumber(Java.java:143)
	at com.atlauncher.utils.Java.getActualJavaVersion(Java.java:153)
	at com.atlauncher.App.main(App.java:318)
```
I didnt test 3.3.0.3, because i didnt have the time but now this comes up; same Problem as in Opening post, Splash shows, but not the Launcher itself. @RyanTheAllmighty
